x-pack/solutions/observability/plugins/apm/common/utils/is_finite_number.ts

```diff
@@ -5,9 +5,6 @@
  * 2.0.
  */
 
-import { isFinite } from 'lodash';
-
-// _.isNumber() returns true for NaN, _.isFinite() does not refine
 export function isFiniteNumber(value: any): value is number {
-  return isFinite(value);
+  return Number.isFinite(value);
```

Comment:
I'm changing this for the following reasons:

1. `Number.isFinite` does the exact same thing, and has been available in all browsers since 2015.
2. [`lodash` is using `Number.isFinite` internally](https://lodash.com/docs/4.17.15#isFinite)
3. Therefore: cut-out the middle-man and just use the native API directly.

Reply:
This wrapping function does a bit of Typescript-magic by asserting the type of `value` being a `number`, which the typing of `Number.isFinite` does not. We are depending on this type behaviour in many places, which would otherwise require `as` casting. Wrapping native methods like this to add additional type behaviour is what you kinda have to do to get this out of Typescript.